ocaml/sdk-gen/component-test/spec/xapi-24/async_vm_import.json

```diff
@@ -0,0 +1,15 @@
+{
+    "xapi-24/async_vm_import_12": {
```

Comment:
Can the number `12` be resorted with the spec file? E.g. it could be `00` or `01` since it is the first one in this spec file.

Reply:
I'm fine if we enforce that there will be only one test case in one spec file. Then there will not be necessary to append numbers to the test cases at all.